### PR TITLE
Remove the PostPermissions middleware

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

**Comment by valo** [comment]
What was this middleware about?

**Comment by IvanIvanoff** [comment]
@valo allowing not logged in users to see only some parts of the insight. For example the content is not available for anonymous users right now

File: test/sanbase_web/graphql/post_test.exs [hunk 1]
```diff
@@ -167,68 +167,68 @@ defmodule SanbaseWeb.Graphql.PostTest do
...
-    }
-    """
-
-    result =
-      conn
-      |> post("/graphql", query_skeleton(query, "allInsights"))
-
-    assert json_response(result, 200)["data"]["allInsights"] == []
-  end
+  # test "trying to get not allowed field from posts as anon user", %{user: user} do
```
Comment (tspenov) [inline test/sanbase_web/graphql/post_test.exs]: lets remove commented tests

Comment (IvanIvanoff) [reply test/sanbase_web/graphql/post_test.exs]: @tspenov I thought we'll need it when reworking the behavior as it won't be like this for long. But I can remove the test and when needed the git history can be searched for it

**Comment by tspenov** [comment]
We need to show only published posts to anonymous users, right ?

**Pull request merged**